week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -71,9 +87,56 @@ function insertShoppingList(shoppingList) {
  * 
  * The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com
  */
+
+ /*const books = [
+  {
+    title: "The Design of Everyday Things",
+    author: "Don Norman",
+    alreadyRead: false
+  },
+  {
+    title: "The Most Human Human",
+    author: "Brian Christian",
+    alreadyRead: true
+  },
+  {
+    title: "The Pragmatic Programmer",
+    author: "Andrew Hunt",
+    alreadyRead: true
+  }
+];*/
+
 function insertBooks(books) {
-  //Write your code in here
+  let images = ["https://encrypted-tbn2.gstatic.com/shopping?q=tbn:ANd9GcTcej0xfvRf7aB-hIonqVwHAgrxSOyUyEAt_z9w1jKmMPudjZG5ZIqZKXqxhmgoBVmmfk3e3b_e4gPkVfKYmn4bAZj6fy9cfL8Rsv7dgi7-&usqp=CAY",
+  "https://images-na.ssl-images-amazon.com/images/I/41m1rQjm5tL._SX322_BO1,204,203,200_.jpg",
+  "https://imagery.pragprog.com/products/595/tpp20.jpg?1557173963"]
+  
+  let content = document.querySelector("#content");
+  let booksList = document.createElement("ul");
+  booksList.className = "ul";
+  content.appendChild(booksList)
+  for (let i = 0; i < books.length ; i ++) {
+    let myBooks = document.createElement("li");
+    myBooks.className = "li";
+    let para = document.createElement("p");
+    let image = document.createElement("img");
+    image.src = images[i];
+    image.width = "260";
+    image.height = "200";
+    myBooks.appendChild(para);
+    myBooks.appendChild(image);
+    para.innerText = books[i].title + " - " + books[i].author;
+    booksList.append(myBooks)
+    booksList.style.display = "flex";
+    booksList.style.margin = "30px";
+    //booksList.style.justifyContent = "space-between"
+    if (books[i].alreadyRead === true) {
+      myBooks.style.backgroundColor = "green"
+    } else { myBooks.style.backgroundColor = "red"}
```

Comment:
Well done here, really nice DOM manipulation but styling could be improved. It looks like the margin should be applied to `li` elements, not to `ul`. You could also do this to get rid of list bullet points:
``` booksList.style.listStyleType = "none";```

It would be nice to give some padding to `li` elements and swap image height and width values so height is `260px` and width `200px`. There is no need to assign className to `ul` and `li` elements if you don’t use them. Images don't have links(anchor tags) to URLs with book covers as per the instruction but you did a good job🙂